### DISCUSSION TITLE
Problemas con la autenticación 2FA

### DISCUSSION THREAD

**Discussion by Susy2021** [community/community#69153] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hola, en reiteradas oportunidades intenté hacer la autenticación 2FA y no me permite la página, temo perder todos mis repositorios. Help Me!!!!

**Comment by ghost** [comment]
To troubleshoot problems with 2FA authentication on GitHub, you can try the following solutions:

Verify that you are following the correct 2FA authentication process. After entering your password, you will need to provide an authentication code. This can be done through a TOTP (Time-based One-Time Password) application like Google Authenticator or through a physical security key. Make sure to follow the instructions provided by GitHub during the 2FA setup process.
If you are using a TOTP application, ensure that you are generating a new authentication code each time you attempt to log in. Some TOTP applications automatically generate a new code every 30 seconds. Check the documentation of your TOTP application for specific instructions on generating a new code.
If you are using a physical security key, ensure that it is properly connected to your device and that you are using it correctly during the authentication process.
If you are receiving an authentication code via SMS, verify that you are receiving the message correctly and entering the code accurately during the authentication process. Make sure that the phone number associated with your GitHub account is correct.
If you are experiencing persistent issues with 2FA authentication, you may need to reach out to GitHub support for further assistance. They will be able to help you troubleshoot specific issues related to your account and 2FA setup.
Remember that 2FA authentication is an additional layer of security to protect your GitHub account. While it can be frustrating to encounter authentication problems, it is important to keep in mind that these measures are designed to safeguard your account and repositories from potential attacks and security breaches.

**Comment by ghostinhershell** [comment]
Hi @Susy2021, thanks so much for reaching out here. Can you clarify the issue that you're running into here? Are you having trouble setting up 2FA for your GitHub account or have you set up 2FA for your GitHub account but you've lost your credentials and can't login? 

Please note that, for security reasons, GitHub Support [will not be able to restore access to accounts](https://docs.github.com/en/site-policy/other-site-policies/github-account-recovery-policy) with two-factor authentication enabled if you lose your two-factor authentication credentials or lose access to your account recovery methods. For more information, see "[Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials)."

**Comment by AwokeKnowing** [comment]
I recommend everyone leave github

They are now just become bullies and another Microsoft app. 

They are unfit to host the open source community's work. Now they don't even let you decide how to authenticate. they force you to tie your account to SMS or some app. 

Move on.  

Github has lost it's way.Forcing 2fa against dev's will is EXACTLY the kind of behavior that made MS infamous. Why kill github? why don't you get it? MAKE 2FA OPTIONAL
